### ISSUE TITLE
storage_download fails( Forbidden (HTTP 403))

### ISSUE THREAD

**Issue by slmbrCat** [Azure/AzureStor#67]
hi,
When I try to get the Blob storage data from the Rstudio Server in my VM (CentOS7), I get the following error massage.
```
Error in download_blob_internal(container, src, dest, blocksize = blocksize,  : 
  Forbidden (HTTP 403). Failed to complete Storage Services operation. Message:

```
Could you please tell me the solution to this problem?
Here's the code I tried

```
library(AzureStor, lib.loc = "/usr/lib64/R/library")
endp <- storage_endpoint("https://mystorageaccount.blob.core.windows.net", key="myaccesskey")
cont <- storage_container(endp, "mycontainer")
storage_download(cont, "myblob.csv", 
                 "myblob_local.csv",
                 overwrite = T)
```
.
sessionInfo()
```
R version 3.6.0 (2019-04-26)
Platform: x86_64-redhat-linux-gnu (64-bit)
Running under: CentOS Linux 7 (Core)

Matrix products: default
BLAS/LAPACK: /usr/lib64/R/lib/libRblas.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] AzureStor_3.2.3

loaded via a namespace (and not attached):
 [1] httr_1.4.2      compiler_3.6.0  R6_2.4.1        AzureRMR_2.3.5 
 [5] tools_3.6.0     curl_4.3        rappdirs_0.3.1  AzureAuth_1.2.4
 [9] mime_0.9        openssl_1.4.2   askpass_1.1    
```

blob storage info:
・ Performance/Access tier：Standard/Hot
・Replication：Read-access geo-redundant storage (RA-GRS)
・ account kind：StorageV2 (general purpose v2)
・ location：Japan East, Japan West
*type of blob is Block blob

VM info
・ OS：Linux (centos 7.8.2003)
・ SKU：centos-76
・ size：Basic A2 (2 vcpus, 3.5 GiB memory)
・ location：Japan East

**Comment by hongooi73** [comment]
Can you try a `list_blobs` or `list_blob_containers`? Also, make sure that you didn't mistype your key.

**Comment by slmbrCat** [comment]
I just ran the code I ran yesterday and it changed to the following error message.
```
Error in curl::curl_fetch_memory(url, handle = handle) : 
  Could not resolve host: mystorageaccount.blob.core.windows.net; Name or service not known
```
The same is true of `list_blobs` and `list_blob_containers`.

![image](https://user-images.githubusercontent.com/56139563/90692722-8ca58280-e2b0-11ea-9149-e58905628bb6.png)

The access keys, account names, etc. are copied from Azure Portal, so they should be correct.
